packages/ethereum/ethereum-types/src/consensus/merkle.rs

```diff
@@ -18,9 +18,9 @@ pub const EXECUTION_PAYLOAD_INDEX: u64 = 25;
 /// The depth of the merkle tree for execution payloads.
 pub const EXECUTION_BRANCH_DEPTH: usize = floorlog2(EXECUTION_PAYLOAD_INDEX);
 /// The depth of the merkle tree for the next sync committee.
-pub const NEXT_SYNC_COMMITTEE_BRANCH_DEPTH: usize = floorlog2(NEXT_SYNC_COMMITTEE_INDEX);
+pub const NEXT_SYNC_COMMITTEE_BRANCH_DEPTH: usize = floorlog2(NEXT_SYNC_COMMITTEE_GINDEX_ELECTRA);
 /// The depth of the merkle tree for the finalized root.
-pub const FINALITY_BRANCH_DEPTH: usize = floorlog2(FINALIZED_ROOT_INDEX);
+pub const FINALITY_BRANCH_DEPTH: usize = floorlog2(FINALIZED_ROOT_GINDEX_ELECTRA);
```

Comment:
The specs don't have these constants. I'm fine with keeping them but I think they should be removed in the long term and we follow the specs more closely